pkg/driver/csi_watch.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 Hewlett Packard Enterprise Development LP
+
+package driver
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	notify "github.com/fsnotify/fsnotify"
+	log "github.com/hpe-storage/common-host-libs/logger"
+)
+
+type CSIWatch struct {
```

Comment:
This seems to be a generic watcher unrelated to CSI.  Can we move this to common-host-libs and rename it to FileWatcher?

Reply:
Yes, this is a generic implementation and doesn't have any CSI specific implementation. This will be useful if at any other place we want to have an event-driven logic. I will move this to common-host-libs. thanks

Reply:
Do you want me to create a separate package FileWatcher inside common-host-libs ? Since none of the existing package names suits watcher implementation.

Reply:
I think it fits in the util package.

Reply:
ok thanks.

Reply:
https://github.com/hpe-storage/common-host-libs/pull/112